app/jobs/mail_forward_job.rb

```diff
@@ -0,0 +1,16 @@
+require 'http'
+# :nocov:
+class MailForwardJob < ApplicationJob
+  queue_as :mail_handlers
+
+  def perform(stored_mail)
+    to_addresses = stored_mail.mail_alias.mail_addresses
+    to_addresses.each do |to_address|
+      # Don't send the mail to the sender
+      next if stored_mail.inbound_email.mail.from.include? to_address
```

Comment:
The sender does not need to receive his own email. In fact, it will likely be marked as spam / untrustworthy